routing/turns_tts_text.cpp

```diff
@@ -62,11 +64,14 @@ string GetTtsText::GetTurnNotification(Notification const & notification) const
   if (notification.m_useThenInsteadOfDistance && notification.m_turnDir == CarDirection::None)
     return string();
 
-  string const dirStr = GetTextById(GetDirectionTextId(notification));
   if (dirStr.empty())
     return string();
 
   string const distStr = GetTextById(GetDistanceTextId(notification));
+
+  if (notification.m_distanceUnits > 0 && !notification.m_nextStreet.empty())
+    return distStr + " " + dirStr + " " + GetTextById("onto") + " " + notification.m_nextStreet;
```

Comment:
We should add "onto" to the sound.txt with all the localizations. And I suspect that it's not an easy task according to languages grammar (not sure that we even can rely on a fixed word's order).